alacritty_terminal/src/term/mod.rs

```diff
@@ -1051,8 +1060,15 @@ impl<T> Term<T> {
             cols.start -= 1;
         }
 
+        let begin_point = self.grid.clamp_buffer_to_visible(Point::new(line, cols.start));
+        let begin_tab_range = self.tabs.tab_range(&self.grid, begin_point);
+
+        // if the range begins in the middle of a tab, move it to the beginning of the tab,
+        // this will include the tab character in the resulting string
+        let col_start = begin_tab_range.map(|range| range.0.start).unwrap_or(cols.start);
```

Comment:
Is the entire reason for this just to prepend a `\t`? Seems like that could be done a little easier.

Reply:
The reason for this is to readjust the start of the selection to the actual beginning of the tab, in order to have the already existing logic for tabs add a `\t` to the copied text.